File: Lista 4/5.py
# 5) Crie um programa que receba o tamanho de três lados de um triângulo e determine se
# as entradas representam um triângulo equilátero, escaleno ou isósceles.

l1, l2, l3 = float(input("Digite o primeiro lado: ")), float(input("Digite o segundo lado: ")), float(input("Digite o terceiro lado: "))

if l1 == l2 and l2 == l3:
    print("Triângulo equilátero.")
elif l1 != l2 and l2 != l3 and l1 != l3:
    print("Triângulo escaleno.")
else:
    print("Triângulo isósceles.")

File: prog/lista4/5.c
// 5) Crie um programa que receba o tamanho de três lados de um triângulo e determine se
// as entradas representam um triângulo equilátero, escaleno ou isósceles.
#include <stdio.h>

int main() {
    float lado1, lado2, lado3;

    scanf("%f %f %f", &lado1, &lado2, &lado3);

    if (lado1 == lado2 && lado2 == lado3) {
        printf("Equilatero.\n");
    } else if (lado1 != lado2 && lado2 != lado3 && lado1 != lado3) {
        printf("Escaleno.\n");
    } else {
        printf("Isosceles.\n");
    }

    return 0;
}